.changeset/cool-news-buy.md

```diff
@@ -0,0 +1,6 @@
+---
+'@urql/preact': patch
+'urql': patch
+---
+
+Clear suspense cache outside of effect for ssr-prepasses and disable caching for concurrent ssr-prepasses.
```

Comment:
```suggestion
Fix server-side rendering by disabling the new Suspense cache on the server-side and clear it for prepasses.
```

Just gonna make this start with that sweet sweet exclamation mark emoji by adding "Fix" 😅